docs/reference/modules/cluster/misc.asciidoc

```diff
@@ -43,6 +43,12 @@ PUT /_cluster/settings
 -------------------------------
 // CONSOLE
 
+NOTE: User-defined cluster metadata is not intended to store sensitive or
+confidential information. Any information stored in user-defined cluster
+metadata will be viewable by anyone with access to the
+<<cluster-get-settings,Cluster Get Settings>> API, and is recorded in the
+Elasticsearch logs.
```

Comment:
I think in recent times our doc writers have been using `{es}` instead of hardcoding "Elasticsearch" so you might want to do the same here.